Microsoft.Toolkit.Services/Services/OneDrive/OneDriveService.cs

```diff
@@ -173,25 +173,6 @@ public bool Initialize(string appClientId, string[] scopes, UIParent uiParent =
             return IsConnected;
         }
 
-        /// <summary>
-        /// Gets the OneDrive root folder (default drive)
-        /// </summary>
-        /// <returns>When this method completes, it returns a OneDriveStorageFolder</returns>
-        public virtual async Task<OneDriveStorageFolder> RootFolderAsync()
```

Comment:
Why did this get removed?

Reply:
Converged authentication sends everyone through RootFolderForMeAsyc() instead. If you use RootFolderAsync() then this works for MSA but fails for School/Work/AD.